doc/ping.xml

```diff
@@ -107,15 +107,27 @@ xml:id="man.ping">
     arbitrary number of “pad” bytes used to fill out the
     packet.</para>
     <para>
+    <command>ping</command> can also send IPv6 Node Information
+    Queries (RFC4620). Intermediate
+    <emphasis remap="I">hop</emphasis>s may not be allowed, because
+    IPv6 source routing was deprecated (RFC5095).</para>
+  </refsection>
+
+  <refsection xml:id="destination">
+    <info>
+      <title>DESTINATION</title>
+    </info>
+    <para>
     <command>ping</command> works with both IPv4 and IPv6. Using
     only one of them explicitly can be enforced by specifying
     <option>-4</option> or
     <option>-6</option>.</para>
     <para>
-    <command>ping</command> can also send IPv6 Node Information
-    Queries (RFC4620). Intermediate
-    <emphasis remap="I">hop</emphasis>s may not be allowed, because
-    IPv6 source routing was deprecated (RFC5095).</para>
+    NOTE: For when doing ping to a IPv6 link-local scope
+    <emphasis remap="I">destination</emphasis>, non-raw ICMP socket
```

Comment:
"non-raw ICMP socket" is more complicated than necessary. There are no "stream icmp sockets" so it's not necessary to say "non-raw". I would say "ICMP datagram socket". That is closer to the [kernel documentation](https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt) for ping_group_range:

> ping_group_range - 2 INTEGERS
> 	Restrict ICMP_PROTO datagram sockets to users in the group range.

Same comment for the message in the second patch.

Reply:
FYI I used that because `non-raw ICMP sockets` term was used already in man page:

https://github.com/iputils/iputils/blob/master/doc/ping.xml#L923-L928

But you're right, thus I use `ICMP datagram socket` as you suggested. Thanks!